src/components/badge/badge.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, h, Prop } from '@stencil/core';
+const ONE_MILION = 1000000;
+const ONE_THOUSAND = 1000;
+const FOUR_DIGIT_LENGTH = 4;
+const FIVE_DIGIT_LENGTH = 5;
+
+@Component({
+    tag: 'limel-badge',
+    styleUrl: 'badge.scss',
+    shadow: true,
+})
+export class Badge {
+    /**
+     * Shows numbers in the badge
+     */
+    @Prop()
+    public badgeCounter: number;
+
+    showRoundedNumber() {
```

Comment:
Did you look into using a library for this? I found this and it looks like it does what we want :slightly_smiling_face: 
https://www.npmjs.com/package/number-abbreviate

Reply:
Nice lib! I think we should override their default units though, because `m` is "milli" not "mega"… (`k`, `M`, `G`, and `T` is definitely less language dependant than `k`, `m`, `b`, and `t`)

Although, to be honest, I wouldn't be surprised if someone from sales or something forces us to use `t` for "tusen" 😩

We'll probably have to enable translations for the units eventually.